recommenders/models/rbm/rbm.py

```diff
@@ -691,3 +699,27 @@ def predict(self, x):
         vp = self.sess.run(v_, feed_dict={self.vu: x})
 
         return vp
+
+    def save(self, dir_name):
+        """Save model parameters in `dir_name`
+        Args:
```

Comment:
Minor change, please add an intro between the first line and Args, otherwise sphinx will throw an error and the documentation won't compile:
```
"""Save model parameters in `dir_name`

        Args:
             dir_name (str): directory name, which should be a folder name instead of file name
                we will create a new directory if not existing.
"""
```

Reply:
Done.